Backend/dev/migrations/dynamic-offer-driver-app/0148-add-ride-location-table-and-columns-to-ride-table.sql

```diff
@@ -0,0 +1,26 @@
+-- before release
+CREATE TABLE atlas_driver_offer_bpp.trip_location as SELECT * FROM atlas_driver_offer_bpp.booking_location;
+ALTER TABLE atlas_driver_offer_bpp.trip_location ADD CONSTRAINT trip_location_pkey PRIMARY KEY (id);
+
+-- before release
+ALTER TABLE atlas_driver_offer_bpp.booking DROP CONSTRAINT IF EXISTS ride_booking_from_location_id_fkey;
+ALTER TABLE atlas_driver_offer_bpp.booking ADD CONSTRAINT ride_booking_from_location_id_fkey FOREIGN KEY (from_location_id) REFERENCES atlas_driver_offer_bpp.trip_location (id);
+ALTER TABLE atlas_driver_offer_bpp.booking DROP CONSTRAINT ride_booking_to_location_id_fkey;
+ALTER TABLE atlas_driver_offer_bpp.booking ADD CONSTRAINT ride_booking_to_location_id_fkey FOREIGN KEY (to_location_id) REFERENCES atlas_driver_offer_bpp.trip_location (id);
+
+-- before release
+ALTER TABLE atlas_driver_offer_bpp.ride ADD COLUMN from_location_id character (36) NOT NULL;
+ALTER TABLE atlas_driver_offer_bpp.ride ADD COLUMN to_location_id character (36) NOT NULL;
```

Comment:
Adding a column and making it not nullable won't work. You can make it not nullable after inserting data.